MatrixKit/Views/ReadReceipts/MXKReadReceiptTableViewCell.m

```diff
@@ -0,0 +1,43 @@
+/*
+ Copyright 2017 Aram Sargsyan
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MXKReadReceiptTableViewCell.h"
+#import "MXKImageView.h"
+
+@implementation MXKReadReceiptTableViewCell
+
+- (void)awakeFromNib
+{
+    [super awakeFromNib];
+    self.avatarImageView.enableInMemoryCache = YES;
+}
+
+- (void)layoutSubviews {
```

Comment:
Please add a new line before starting a new block {}. Like you did for the other blocks